plugins/discourse-presence/assets/javascripts/discourse/lib/presence-manager.js.es6

```diff
@@ -200,4 +214,73 @@ const PresenceManager = EmberObject.extend({
   }
 });
 
+const PresenceManager = EmberObject.extend({
+  presences: null,
+  currentUser: null,
+  messageBus: null,
+  siteSettings: null,
+
+  init() {
+    this._super(...arguments);
+
+    this.setProperties({
+      presences: {}
+    });
+  },
+
+  subscribe(topicId, type) {
+    if (!topicId) return;
```

Comment:
Not sure how I can get around having to litter this every where to handle the case where the composer does not have a `topic.id`. Mainly when creating new topics.